### PR TITLE
Added remaining color types for PNTS

### PR DESCRIPTION
**Pull request by pizza3**
Added support for rgba, rgb565 and constant_rgba for PNTSLoader.

The example data used for [rgb](https://github.com/CesiumGS/cesium/tree/main/Specs/Data/Cesium3DTiles/PointCloud/PointCloudRGBA) [rgb565](https://github.com/CesiumGS/cesium/tree/main/Specs/Data/Cesium3DTiles/PointCloud/PointCloudRGB565) [constant_rgba](https://github.com/CesiumGS/cesium/tree/main/Specs/Data/Cesium3DTiles/PointCloud/PointCloudConstantColor)

### CODE REVIEW

File: src/three/PNTSLoader.js [hunk 1]
```diff
@@ -111,20 +116,48 @@ export class PNTSLoader extends PNTSLoaderBase {
...
+					}
+					geometry.setAttribute( 'color', new BufferAttribute( color, 3, true ) );
+					material.vertexColors = true;
+
+				} else if ( CONSTANT_RGBA !== null ) {
+
+					const color = new Color( `rgb(${CONSTANT_RGBA[ 0 ]}, ${CONSTANT_RGBA[ 1 ]}, ${CONSTANT_RGBA[ 2 ]})` );
+					material.color = color;
+					material.opacity = CONSTANT_RGBA[ 3 ] / 255;
+					material.transparent = true;
```
Comment (gkjohnson) [inline src/three/PNTSLoader.js:153]: This should only be set to true if the opacity is < 1.0

File: src/utilities/rgb565torgb.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export function rgb565torgb( rgb565 ) {
+
+	let r = ( rgb565 & 0xF800 ) >> 11;
+	let g = ( rgb565 & 0x07E0 ) >> 5;
+	let b = rgb565 & 0x001F;
+	r = ( r << 3 ) | ( r >> 2 ); // Scale up red component
+	g = ( g << 2 ) | ( g >> 4 ); // Scale up green component
+	b = ( b << 3 ) | ( b >> 2 ); // Scale up blue component
```
Comment (gkjohnson) [inline src/utilities/rgb565torgb.js:8]: These all seem like magic numbers. Does this 565 unpacking implementation come from somewhere?

**Review by gkjohnson — COMMENTED** [review]
Thanks! A couple comments - I'll give this a test after those are handled.

File: src/utilities/rgb565torgb.js (hunk 2 above)
Comment (pizza3) [reply src/utilities/rgb565torgb.js]: Hey, I updated the 565 unpacking to a much cleaner one I found over here [link](https://rgbcolorpicker.com/565), the previous one was taken from a arduino forum.

Comment (gkjohnson) [reply src/utilities/rgb565torgb.js]: Great, thanks! Can we add a comment with a link to that page - then I'll test it and get this merged!

Comment (pizza3) [reply src/utilities/rgb565torgb.js]: ok 👍

**Comment by gkjohnson** [comment]
Made a few small changes - but this looks great! Thanks.

**Pull request merged**